assets/js/components/help/HelpMenu.js

```diff
@@ -83,15 +83,16 @@ function HelpMenu( { children } ) {
 					ref={ menuRef }
 					menuOpen={ menuOpen }
 					id="googlesitekit-help-menu"
+					onSelected={ () => {} }
```

Comment:
Instead of adding an empty function here, let's better add it as a default prop in the Menu component itself.

Reply:
👍